### ISSUE TITLE
Missing/conflicting dependency

### ISSUE THREAD

**Issue by dgoebel97** [ArunaStorage/python-api#4]
When executing the example code in the documentation after a fresh install in a virtual environment with `pip install ArunaP-ython-API==2.0.0b9`, python throws the exception 
```
ModuleNotFoundError: No module named 'protoc_gen_openapiv2'
```
While installing the missing dependency with `pip install protoc_gen_openapiv2`, pip complains about a conflicting dependency.
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
aruna-python-api 2.0.0b9 requires protobuf==3.20.3, but you have protobuf 4.25.1 which is incompatible.
```
The module gets still installed and the example code can be executed.

**Comment by das-Abroxas** [comment]
Thank you for reporting this issue.

The problem is already known and will be fixed in the next release by integrating the generated _protoc_gen_openapiv2_ imports into the package. Until then, the two steps (which you have already described) work as a workaround:

1. Update _protobuf_: `pip install --upgrade protobuf`
2. Install package which provides the missing module: e.g. `pip install grpc-protoc-annotations` or `pip install protoc_gen_openapiv2`

The error message that _protobuf_ version 4.25.1 is incompatible can be ignored, as it is only generated by the entry in _setup.py_ `install_requires=['protobuf==3.20.3', ...` which is no longer correct. 

I will keep the issue open for documentation until the next release, in case others also encounter this problem.

**Comment by databaseJena** [comment]
> Thank you for reporting this issue.
> 
> The problem is already known and will be fixed in the next release by integrating the generated _protoc_gen_openapiv2_ imports into the package. Until then, the two steps (which you have already described) work as a workaround:
> 
> 1. Update _protobuf_: `pip install --update protobuf`
> 2. Install package which provides the missing module: e.g. `pip install grpc-protoc-annotations` or `pip install protoc_gen_openapiv2`
> 
> The error message that _protobuf_ version 4.25.1 is incompatible can be ignored, as it is only generated by the entry in _setup.py_ `install_requires=['protobuf==3.20.3', ...` which is no longer correct.
> 
> I will keep the issue open for documentation until the next release, in case others also encounter this problem.

It works for me while previously I tried and failed. I read it again and noticed one type is here, `pip install --upgrade protobuf`

Thanks
(Linux ubuntu server 22.04, conda environment, python3)

**Comment by das-Abroxas** [comment]
We have released version `2.0.0-beta.10`.
With this version the Aruna API for Python can be installed again without extra steps via: `pip install Aruna-Python-API==2.0.0b10`.

In the foreseeable future, version 2.0.0 of the Aruna API will finally be released as a stable version, which will also eliminate the need to specify the pre-release version.
